homeassistant/components/zha/__init__.py

```diff
@@ -185,5 +185,14 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         config_entry.version = 3
         hass.config_entries.async_update_entry(config_entry, data=data)
 
+    if config_entry.version == 3:
```

Comment:
These bumps are backward incompatible, is that really needed to do for a change like this?

Alternatively, we can also just handle this when setting up the connection.

../Frenck

Reply:
Can we update a config flow without bumping up the schema?

Reply:
Yes, that is possible to, check the value of the config data on config entry setup and adjust it if in case it isn't correct. Keep it around for some cycles and it should be good 👍

Reply:
Thanks! This now is the current behavior.